modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -27,28 +28,50 @@ local function new_{{classname}}(host, basePath, schemes)
     basePath = basePath or "{{{basePath}}}";
     schemes = schemes_map;
     default_scheme = default_scheme;
+    http_username = '';
```

Comment:
Sure. I was worried it would cause issues later when doing string concatenation.